src/cr/cube/matrix/assembler.py

```diff
@@ -342,6 +342,17 @@ def pairwise_significance_means_t_stats(self, column_idx):
             self._measures.pairwise_significance_means_t_stats(base_column_idx).blocks
         )
 
+    @lazyproperty
+    def cell_population_fractions(self):
+        proportions = (
+            self.row_proportions
+            if self._rows_dimension.dimension_type in (DT.CAT_DATE, DT.CA_SUBVAR)
+            else self.column_proportions
+            if self._columns_dimension.dimension_type in (DT.CAT_DATE, DT.CA_SUBVAR)
+            else self.table_proportions
+        )
+        return proportions
```

Comment:
Okay, needs a docstring but I see what you're doing here. Maybe `.population_proportions`. I would interpret this as the fraction of the overall population represented by the base for each cell.

We've historically used the prefix `cell_` to mean the same concept as `table_` means here, like `cell_percents` probably still lives somewhere in exporter somewhere and meaning `table_` (or perhaps some places `total_`) percents. So `cell_` is probably better avoided as potentially confusing, also all proportions measures have a value for each cell, so no need to be explicit about that.

I wouldn't involve a new `proportions` name here, `return (self.row_ ... if ... else ...)` states it clearly and simply to me. The docstring can fill in the idea that the proportions basis varies and why.

Reply:
Btw, this property is out of position would belong much higher up in alphabetical order. Interestingly, if you change the name to `.population_proportions` then it's in exactly the right position.